client/knode/validator/validator.go

```diff
@@ -36,6 +36,11 @@ var (
 
 var (
 	txConfirmationTimeout = 10 * time.Second
+
+	// log handlers
+	fileHandler     = log.Must.FileHandler("/var/log/validator_all.log", log.LogfmtFormat())
```

Comment:
It can break non-unix clients or clients with no access to /var/log

Reply:
agree, this needs to be configurable and passed down from main comand